vendors/espressif/boards/ports/ota_pal_for_aws/aws_esp_ota_ops.h

```diff
@@ -0,0 +1,135 @@
+// Copyright 2015-2016 Espressif Systems (Shanghai) PTE LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _AWS_ESP_OTA_OPS_H
```

Comment:
Same question with this as the .c file. How come this file changed?

Reply:
Additional helper functions were introduced during previous iterations.